bots/github/cache.py

```diff
@@ -48,10 +48,13 @@ def prune(self):
         if not os.path.exists(self.directory):
             return
         now = time.time()
-        for filename in os.listdir(self.directory):
-            path = os.path.join(self.directory, filename)
-            if os.path.isfile(path) and os.stat(path).st_mtime < now - 7 * 86400:
-                os.remove(path)
+        try:
+            for filename in os.listdir(self.directory):
+                path = os.path.join(self.directory, filename)
+                if os.path.isfile(path) and os.stat(path).st_mtime < now - 7 * 86400:
+                    os.remove(path)
+        except OSError:
+            pass
```

Comment:
Oh right, they might get ripped out underneath the  loop for stat() too, sorry for not seeing that earlier. Seems okay to me then.